ci-operator/step-registry/rosa/dnsdomain/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
I think this feature is used for shared-vpc which is only available for the rosa STS cluster. If yes, let us put this under the directory `rosa/sts/dnsdomain` too.

Reply:
@yasun1  I thought this step is independency of any kind of cluster (create/delete DNS domain on server side using ROSA cli), at present, it's only used by Shared-VPC cluster, but it might be used in another type of cluster in the future. 
But if you think it's better to put it under `sts` folder, let me know I will update.

Reply:
For the account-roles side, the cluster types are `sts` and `non-sts`. In your script, the dnsdomain requires the installer-role, that is why I say this should be a STS step.

Reply:
I don't see any installer role is required.